packages/vite/src/node/plugins/importAnalysisBuild.ts

```diff
@@ -161,7 +161,11 @@ export function buildImportAnalysisPlugin(config: ResolvedConfig): Plugin {
         }
       }
 
-      if (needPreloadHelper && !ssr) {
+      if (
+        needPreloadHelper &&
+        !ssr &&
+        source.indexOf(`const ${preloadMethod} =`) < 0
```

Comment:
We can use `includes` here
```suggestion
        !source.includes(`const ${preloadMethod} =`)
```